common.go

```diff
@@ -24,22 +27,32 @@ const (
 	VarSAggregation  AggregationType = "VAR.S"
 )
 
+const (
+	BlockDuplicatePolicy DuplicatePolicyType = "block" // an error will occur for any out of order sample
+	FirstDuplicatePolicy DuplicatePolicyType = "first" // ignore the new value
+	LastDuplicatePolicy  DuplicatePolicyType = "last"  // override with latest value
+	MinDuplicatePolicy   DuplicatePolicyType = "min"   // only override if the value is lower than the existing value
+	MaxDuplicatePolicy   DuplicatePolicyType = "max"   // only override if the value is higher than the existing value
+)
+
 var aggToString = []AggregationType{AvgAggregation, SumAggregation, MinAggregation, MaxAggregation, CountAggregation, FirstAggregation, LastAggregation, StdPAggregation, StdSAggregation, VarPAggregation, VarSAggregation}
 
 // CreateOptions are a direct mapping to the options provided when creating a new time-series
 // Check https://oss.redislabs.com/redistimeseries/1.4/commands/#tscreate for a detailed description
 type CreateOptions struct {
-	Uncompressed   bool
-	RetentionMSecs time.Duration
-	Labels         map[string]string
-	ChunkSize      int64
+	Uncompressed    bool
+	RetentionMSecs  time.Duration
+	Labels          map[string]string
+	ChunkSize       int64
+	DuplicatePolicy DuplicatePolicyType
 }
 
 var DefaultCreateOptions = CreateOptions{
-	Uncompressed:   false,
-	RetentionMSecs: 0,
-	Labels:         map[string]string{},
-	ChunkSize:      0,
+	Uncompressed:    false,
+	RetentionMSecs:  0,
+	Labels:          map[string]string{},
+	ChunkSize:       0,
+	DuplicatePolicy: "",
```

Comment:
@danni-m I believe it should remain as an empty strubg, given that RedisTimeSeries also returns a null string on ts.info. My main goal was to avoid any defaults on the client side and only issue the duplicate policy when it is meaningfully set. WDYT?

Reply:
@filipecosta90 makes sense. the risk here is that this will send an empty string as the default policy if the user didn't provide one.

Reply:
@danni-m we're explicitly checking for it here:
https://github.com/RedisTimeSeries/redistimeseries-go/pull/84/files#diff-36caaa1394a5803bfa9f964755f1d733R108

Reply:
👍